tests/jobsautoreport/test_slack.py

```diff
@@ -0,0 +1,58 @@
+from datetime import timedelta
+from unittest.mock import MagicMock
+
+from jobsautoreport.slack import SlackReporter
+
+
+def test_send_message():
```

Comment:
```suggestion
def test_send_report_should_succeessfully_call_slack_api_with_expected_message_format():
```
or something like that